examples/filesystem/README.md

```diff
@@ -7,16 +7,21 @@ application.
 
 In particular, this example shows:
 
-- how to mount/format/unmount a file system, either with spiffs, littlefs, fatfs
-  or constfs
+- how to mount/format/unmount a file system, either with constfs for all
+  devices and the default fs for devices that have a filesystem mountpoint
+  configured by the board.
 - how to open/read/write/close a file with and without newlib
 
-In RIOT, most file systems use a `mtd` as flash interface. So to use this
-example one must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t`
-instance.
+A `constfs` file system is demonstrated with two files for all targets
+to demonstrate the file system functionality without an external mass
+storage device.
+
+For targets that feature an onboard external mass storage device with a
+mount point configured, like the Nordic nRF52840 Development Kit
+(see `boards/nrf52840dk`), the flash chip is automatically added by VFS.
 
 This example uses `littlefs` as default file system on the whole `mtd`.
```

Comment:
You can drop this line

Reply:
The whole "For targets ... added by VFS."?
I would perhaps suggest to leave out the nRF example, and change it to 

```For targets that feature an onboard external mass storage device with a
mount point configured, the flash chip is automatically added by VFS.```

For me it was a very important hint, that this is the case, because otherwise you don't know where to look for the MTD definitions (in case the development board does not come by default with a flash and you want to add one, which is what I did).

Reply:
No I mean

> This example uses `littlefs` as default file system on the whole `mtd`.